pyvista/utilities/reader.py

```diff
@@ -118,20 +120,42 @@ class BaseReader:
     def __init__(self, filename):
         """Initialize Reader by setting filename."""
         self._reader = self._class_reader()
+        if not os.path.isfile(filename):
+            raise FileNotFoundError(f'{filename} does not exist.')
```

Comment:
This is probably okay, but some readers can sometimes take a folder.  We may need to override or bypass this method in that case.